src/pages/SubmitWorkflow.js

```diff
@@ -41,7 +43,16 @@ export const SubmitWorkflow = () => {
 
   const submitRun = async () => {
     try {
-      await Ajax(signal).Cbas.runs.post(workflowUrl, workflowInputs)
+      const runSetsPayload = {
+        workflow_url: workflowUrl,
+        workflow_param_definitions: JSON.parse(workflowInputsDefinition),
+        wds_entities: {
+          entity_type: entityType,
+          entity_ids: JSON.parse(entityId)
```

Comment:
Curious why the entityId needs parsing first?

Reply:
Without the parsing, the array was passed as an escaped String and CBAS would return 400 Bad Request